contracts/settling_game/L04_Calculator.cairo

```diff
@@ -58,6 +58,24 @@ func constructor{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_p
     return ()
 end
 
+@view
+func calculateEpoch{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (epoch : felt):
```

Comment:
This looks quite similar to `get_current_epoch` that's added [here](https://github.com/BibliothecaForAdventurers/realms-contracts/pull/23/files#diff-3b3dc2cb8f0921bef3f1fc9082a786e0f21d4de782722e515adffbda9c815bd0R143).